src/Uno.UWP/FeatureConfiguration/WinRTFeatureConfiguration.Focus.cs

```diff
@@ -1,14 +1,15 @@
-﻿namespace Uno;
-
-partial class WinRTFeatureConfiguration
+﻿namespace Uno
 {
```

Comment:
Why is this moving from file-scoped namespaces to block namespaces?

Reply:
Whoopsie!